### ISSUE TITLE
Add `libcuda.so.1` to the list of lib names?

### ISSUE THREAD

**Issue by agerasev** [coreylowman/cudarc#279]
I have an Ubuntu 22.04.4 LTS machine with CUDA driver 550.90.07 installed.

There is no `libcuda.so` or `libnvcuda.so`, but only these files:
```
/usr/lib/x86_64-linux-gnu/libcuda.so.550.90.07
/usr/lib/x86_64-linux-gnu/libcuda.so.1
```
where the second is a symlink to the first.

When I made another symlink named `libcuda.so` everything works fine.

Maybe `libcuda.so.1` should be added to the list of library names?

**Comment by coreylowman** [comment]
Hey this was just added a few weeks ago but not released to cargo - will release shortly. Closing this for now

**Comment by CatsAreFluffy** [comment]
`libcuda.so.1` is still not detected. If it was, it'd fix #296.
